copyright.txt

```diff
@@ -0,0 +1,998 @@
+       ui/pixmaps/xopt.svg
+Copyright: 2020, Luya Tshimbalanga <luya@fedoraproject.org>
+License: CC-BY-SA-4.0
+Comment: See https://github.com/xournalpp/xournalpp/issues/3103#issuecomment-844556592
+
+Files: cmake/find/FindFilesystem.cmake
+       cmake/include/clang-tidy.cmake
+Copyright: 2000-2021, Kitware, Inc. and Contributors
+License: BSD-3-clause
+
+Files: cmake/find/FindICU.cmake
+Copyright: 2011-2013, julp
+License: BSD-3-clause
+Comment: https://github.com/julp/FindICU.cmake
+         could maybe be replaced with
+         https://cmake.org/cmake/help/latest/module/FindICU.html
+
+Files: cmake/include/Copyright.txt
+Copyright: 2015, Marek Pikuła <marpirk@gmail.com>
+           Xournal++ team
+License: BSD-3-clause
+
+Files: cmake/include/ExternalDeb.cmake
+       cmake/include/GitRepo.cmake
+Copyright: 2015, Marek Pikuła <marek@pikula.co>
+License: BSD-3-clause
+
+Files: cmake/find/Gettext.cmake
+Copyright: 2012-2013, Valama development team
+License: GPL-3+
+
+Files: desktop/com.github.xournalpp.xournalpp.appdata.xml
+Copyright: 2019, Tobias Mueller <tobiasmue@gnome.org>
+License: CC0-1.0
+
+Files: src/util/hashcombine.h
+Copyright: 2019, Fabian Keßler <fabian_kessler@gmx.de>
+           1998-2021, David Eberly, Geometric Tools
+License: BSL-1
+
+Files: debian/docs
+       debian/package_description
+       debian/control
+       debian/changelog
+       debian/compat
+       debian/rules
+       debian/source/format
+Copyright: 2007, Mathieu Bouchard <mbouchar@bioinfo.ulaval.ca>
+           2008, Lucas Nussbaum <lucas@lucas-nussbaum.net>
+           2009-2014, Carlo Segre <segre@debian.org>
+           2018-2019, Andreas Butti <andreas.butti@gmail.com>
+           2020-2021, Barak A. Pearlmutter <bap@debian.org>
+License: GPL-2+
+
+Files: debian/copyright
+Copyright: 2018, Andreas Butti <andreasbutti@gmail.com>
+License: GPL-2+
+
+Files: plugins/MigrateFontSizes/plugin.ini
+Copyright: 2021, Roland Lötscher
+License: GPL-2+
+
+Files: plugins/ColorCycle/plugin.ini
+       plugins/LayerActions/plugin.ini
+Copyright: 2020, Roland Lötscher
+License: GPL-2+
+
+Files: plugins/Example/plugin.ini
+       plugins/ToggleGrid/plugin.ini
+Copyright: 2019, Andreas Butti
+License: GPL-2+
+
+Files: plugins/QuickScreenshot/plugin.ini
+Copyright: 2020, Debbie Reynolds
+License: GPL-2+
+
+Files: src/gui/GladeSearchpath.h
+Copyright: 2011, Andreas Butti
+License: GPL-2+
+
+Files: src/gui/dialog/LatexDialog.h
+Copyright: 2013, Wilson Brenna
+License: GPL-2+
+
+Files: src/util/pixbuf-utils.cpp
+Copyright: 2001, 2002, 2003, The Free Software Foundation, Inc.
+           2003, Ettore Perazzoli
+           1995, Spencer Kimball and Peter Mattis
+License: GPL-3+
+Comment: Original Author Paolo Bacchilega <paolo.bacch@tin.it>
+          seemingly for the FSF but this is unclear from the code
+License: GPL-3+
+
+Files: src/util/safe_casts.h
+Copyright: August 17th, 2003, Fabian Keßler <fabian_kessler@gmx.de>
+License: BSL-1
```

Comment:
Hehe in 2003 I was in the elementary school so that date can't be true.

Reply:
Way to start early :wink:  I checked the commit and updated it to 2020.

Reply:
No he also copied it(but he created the file and referenced the cited paper). Maybe it is ok to give the Copyright of that file to me, since I created it and I just cited the code from boost, so we need to cite the boost source?

And the one who created the boost file is Daniel James (2005-2014)